catalog/rocks/entry_catalog.go

```diff
@@ -65,14 +65,19 @@ type EntryCatalog struct {
 	store graveler.Graveler
 }
 
+const (
+	RangeFSName     = "range"
+	MetaRangeFSName = "meta-range"
```

Comment:
1. We don't need these defaults in graveler?
2. Didn't discuss it - but we need to choose between metarange and meta-range?

Reply:
1. Nope - You only need them to create the FS. Diagnostics needs them to read the blockstorage prefix.
2. Didn't get what you meant.

Reply:
1. cool
2. there are places (not in your code) where we call it "metarange" and places "meta-range" :)